vscripts/lovol_fix.nut

```diff
@@ -0,0 +1,1110 @@
+		EntFire("lvl4item_temp","Trigger","",3.00,null);
+		EntFire("lvl4item_position","Trigger","",3.00,null);
+		EntFire("stage_","SetTextureIndex","6",0.00,null);
+		EntFire("lvl4_item_toilet2","PickRandom","",0.7,null);
+		EntFire("lvl4_item_toilet1","PickRandom","",0.5,null);
+		EntFire("l0v0l_fog","SetStartDist","600",0.2,null);
+		EntFire("l0v0l_fog","SetEndDist","2500",0.2,null);
+		EntFire("l0v0l_fog","SetColor","0 0 0",0.2,null);
+		EntFire("torchlvl2","Start","",0.5,null);
+		EntFire("l0v0l_fog","TurnOn","",0.00,null);
+		EntFire("map_command","Command","say >>> Map made by Moray <<<",4.00,null);
+		EntFire("bgm_lvl4first","PlaySound","",3.00,null);
+		EntFire("map_command","Command","say >>> Special thanks to hobbit <<<",5.00,null);
+	}
+	EntFireByHandle(self, "RunScriptCode", " RestartParticle(); ", 0.00, null, null);
+	EntFireByHandle(self, "RunScriptCode", " KillEnt(); ", 0.00, null, null);
+}
+
+function RestartParticle()
+{
+    EntFire("spawnlight","Stop","",3.00,null);
+	EntFire("spawnlight","Start","",5.00,null);
+	EntFire("teleport_effnormal","Stop","",3.00,null);
+	EntFire("teleport_effnormal","Start","",5.00,null);
+	EntFire("waterfall_2","Stop","",3.00,null);
+	EntFire("waterfall_2","Start","",5.00,null);
+	EntFire("waterfall_3","Stop","",3.00,null);
+	EntFire("waterfall_3","Start","",5.00,null);
+	EntFire("cliff_push","Stop","",3.00,null);
+	EntFire("cliff_push","Start","",5.00,null);
+	EntFire("heal_eff","DestroyImmediately","",3.00,null);
+	EntFire("heal_eff","Start","",5.00,null);
+   	EntFire("ammo_eff","DestroyImmediately","",3.00,null);
+	EntFire("ammo_eff","Start","",5.00,null);
+	EntFire("zgravity_eff","DestroyImmediately","",3.00,null);
+	EntFire("zgravity_eff","Start","",5.00,null);
+   	EntFire("zheal_eff","DestroyImmediately","",3.00,null);
+	EntFire("zheal_eff","Start","",5.00,null);
+	EntFire("beam_eff","DestroyImmediately","",3.00,null);
+	EntFire("beam_eff","Start","",5.00,null);
+	EntFire("wind_eff","DestroyImmediately","",3.00,null);
+	EntFire("wind_eff","Start","",5.00,null);
+	EntFire("poison_eff","DestroyImmediately","",3.00,null);
+	EntFire("poison_eff","Start","",5.00,null);
+	EntFire("firet_eff","DestroyImmediately","",3.00,null);
+	EntFire("firet_eff","Start","",5.00,null);
+	EntFire("icet_eff","DestroyImmediately","",3.00,null);
+	EntFire("icet_eff","Start","",5.00,null);
+   	EntFire("elect_eff","DestroyImmediately","",3.00,null);
+	EntFire("elect-eff","Start","",5.00,null);
+	EntFire("poisont_eff","DestroyImmediately","",3.00,null);
+	EntFire("poisont_eff","Start","",5.00,null);
+   	EntFire("gravityt_eff","DestroyImmediately","",3.00,null);
+	EntFire("gravityt_eff","Start","",5.00,null);
+	EntFire("eartht_eff","DestroyImmediately","",3.00,null);
+	EntFire("eartht_eff","Start","",5.00,null);
+   	EntFire("watert_eff","DestroyImmediately","",3.00,null);
+	EntFire("watert_eff","Start","",5.00,null);
+	EntFire("pusht_eff","DestroyImmediately","",3.00,null);
+	EntFire("pusht_eff","Start","",5.00,null);
```

Comment:
None of this is necessary, you've already delayed the particle starts in MapStart.

Realistically the porter could've done the same thing and avoided needing this function at all.